osidb/models.py

```diff
@@ -592,10 +593,54 @@ def __str__(self):
         """return cve_id when str(flaw)"""
         return str(self.cve_id)
 
+    def _validate_rh_nvd_cvss_score_diff(self):
+        """
+        Checks that the difference between the RH and NVD CVSS score is not >= 1.0
+        """
+        if not self.cvss3_score or not self.nvd_cvss3:
```

Comment:
I am not sure, if you should check the `cvss3_score` directly. It is a float value and potentially it can be equal to `0.0`, in that case you are skipping the check even if the scores exists. You should probably either check it for `is not None` or check rather `cvss3`.

Reply:
Good catch, we probably should have an extra validation that checks that `cvss3_score` matches the value from `cvss3`, because at the moment if you create a flaw from the API you can provide one and not the other, or maybe it should be a computed value altogether... that said don't know if it's worth the effort or if we should just wait until we refactor CVSS into a model

Reply:
It should be ideally tied together and not independent so it can never desync. But maybe let's wait until the refactor.